Program.cs

```diff
@@ -42,4 +42,4 @@
 // end
 Console.WriteLine("");
 Console.Write("Press Any Key to Continue...");
-Console.ReadKey();
```

Comment:
I don't know why it's saying I removed this line, I didn't.

Reply:
its saying you added a new line smh

Reply:
> its saying you added a new line smh

Yes but I didn't!